package.json

```diff
@@ -1,12 +1,12 @@
 {
-  "name": "grunt-html",
+  "name": "grunt-html-2",
```

Comment:
Yes, I needed this to add to npm so I could get it into a project. I'll have this removed.